src/init/WorkerManager.ts

```diff
@@ -0,0 +1,20 @@
+import type { ClusterManager } from './ClusterManager';
+import { Initializer } from './Initializer';
+
+/**
+ * Creates and starts an HTTP server.
```

Comment:
I'm assuming this class is unfinished?

Reply:
JSDoc needed updating, but implementation is just that: spawning necessary workers when CSS is started in multithreaded mode.

Reply:
Ah yes, for some reason I read it that `ServerInitializer` was removed, but it was just moved to the worker initializer.